src/azure-cli/azure/cli/command_modules/util/custom.py

```diff
@@ -140,6 +140,13 @@ def upgrade_version(cmd, update_all=None, yes=None):  # pylint: disable=too-many
                            "or run 'pip install --upgrade azure-cli' in this container")
         elif installer == 'MSI':
             _upgrade_on_windows()
+        elif installer == 'ZIP':
+            if platform.architecture()[0] == '32bit':
+                zip_url = 'https://aka.ms/installazurecliwindowszip'
+            else:
+                zip_url = 'https://aka.ms/installazurecliwindowszipx64'
+            logger.warning("Please download the latest ZIP from %s, delete the old installation folder and extract the "
+                           "new version to the same location", zip_url)
```

Comment:
When extracting a folder with the same name, the default action is to merge, which can lead to unexpected issues.
Deleting the old folder is required.